6.x/base-how-to.md

```diff
@@ -413,3 +413,71 @@ Add whatever validation rules & inputs you want, in addition to name and passwor
 ```
 This will make the registration process pick up a view you can create, in ```resources/views/vendor/backpack/ui/auth/register.blade.php```. You can copy-paste the original view, and modify as you please. Including adding your own custom inputs.
 
+<a name="enable-email-verification-in-backpack-routes"></a>
+### Enable email verification in Backpack routes
+
+In Backpack CRUD 6.2 we introduced the ability to require email verification when accessing Backpack routes.
+
+To enable this feature please do the following: 
```

Comment:
Too many paragraphs. It's one idea, so it doesn't need two paragraphs. We should avoid having short but many paragraphs - it does not read easily, and it does not look good. The rule of thumb is one-idea-one-paragraph. Sometimes it's not possible and we can split one idea into multiple paragraphs. But only when each paragraph is "long enough". If paragraphs are short and the same idea, they need squishing 😀

```suggestion
In Backpack CRUD 6.2 we introduced the ability to require email verification when accessing Backpack routes. To enable this feature please do the following: 
```